functional-tests/tests/test_anonymizer.py

```diff
@@ -24,7 +24,8 @@ def test_given_anonymize_called_with_valid_request_then_expected_valid_response_
 
     response_status, response_content = anonymize(request_body)
 
-    expected_response = "hello world, my name is ANONYMIZED. My number is: 03445****"
+    expected_response = '{"text":"hello world, my name is ANONYMIZED. ' \
```

Comment:
are'nt we using "response" and not "text"?

Reply:
Correct, will be fixed.